{{cookiecutter.project_slug}}/src/store/index.js

```diff
@@ -0,0 +1,28 @@
+import { apiReducer } from "./reducers";
+import { combineReducers } from "redux";
+import { createStore, applyMiddleware, compose } from "redux";
```

Comment:
combineReducers should be in one import line e.g. line 3

Reply:
Fixed in cab7061